src/plugins/telemetry_management_section/public/components/__snapshots__/telemetry_management_section.test.tsx.snap

```diff
@@ -57,13 +57,13 @@ exports[`TelemetryManagementSectionComponent renders as expected 1`] = `
           loading={false}
           setting={
             Object {
-              "ariaName": "Provide usage data",
+              "ariaName": "Share usage with Elastic",
               "category": Array [],
               "defVal": true,
               "description": <React.Fragment>
                 <p>
                   <FormattedMessage
-                    defaultMessage="Enabling data usage collection helps us manage and improve our products and services. See our {privacyStatementLink} for more details."
+                    defaultMessage="Enabling usage collection (also known as Telemetry) allows us to learn what our users are most interested in, so we can improve our products and services. Refer to our {privacyStatementLink} for more details."
```

Comment:
Remove "for more details"?   It isn't in the screenshots

Reply:
It is in the **Advanced settings**. I'll remove it anyway :)